### PR TITLE
Rob Ede's interview from Mainmatter's Rust interview series #226

### PR DESCRIPTION
**Pull request by LukeMathWalker**

### CODE REVIEW

File: _episodes/interviews/2023-10-18-rob-ede.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Kraken's migration to Rust microservices, with Rob Ede"
+date: 2023-10-18T19:46:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-10-18-rob-ede.mp3
```
Comment (jonhoo) [inline _episodes/interviews/2023-10-18-rob-ede.md:4]: Sorry it took me so long to get to this! I've uploaded the mp3 as 2023-10-19-. Can you update this as well as the name of the `.md` file?

File: _episodes/interviews/2023-10-18-rob-ede.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+duration: "32:13"
+length: "34292702"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+[Rob Ede](https://github.com/sponsors/robjtede), lead maintainer of `actix-web`, explains to Marco Otto-Witte how (and why) Kraken chose to migrate their microservices to Rust.
```
Comment (jonhoo) [inline _episodes/interviews/2023-10-18-rob-ede.md:10]: I'd like a link to both `actix-web` and Kraken here

File: _episodes/interviews/2023-10-18-rob-ede.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+- [@00:00] - Introduction
+    - [Video recording of the interview](https://mainmatter.com/blog/2023/09/06/krakens-migration-to-rust-microservices/)
+- [@00:36] - Start of the interview
+- [@01:26] - What is `actix-web`?
```
Comment (jonhoo) [inline _episodes/interviews/2023-10-18-rob-ede.md:28]: Link

File: _episodes/interviews/2023-10-18-rob-ede.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- [@06:34] - Kraken's migration from Java to Rust
```
Comment (jonhoo) [inline _episodes/interviews/2023-10-18-rob-ede.md:29]: Link

**Review by jonhoo — CHANGES_REQUESTED** [review]
Sorry for the delay! Just a few tweaks and then we can get it out tomorrow :+1:

**Comment by LukeMathWalker** [comment]
I _think_ I got them all 👍🏻

**Review by jonhoo — APPROVED** [review]
Thanks! Some of the links could just have been sub bullets, but I'll take it 😅

File: _episodes/interviews/2023-10-19-rob-ede.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Kraken's migration to Rust microservices, with Rob Ede"
+date: 2023-10-19T19:46:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-10-19-rob-ede.md:3]:
```suggestion
date: 2023-10-19T16:57:00Z
```

**Pull request merged**